install/kubernetes/helm/istio/charts/kiali/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kiali
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: kiali
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  {{ .Values.replicaCount }}
```

Comment:
should be: `replicas: {{ .Values.replicaCount }}`

Reply:
Done